tests/ts_A_select_compare_two_fields.erl

```diff
@@ -1,4 +1,4 @@
--module(ts_A_select_fail_5).
+-module(ts_A_select_compare_two_fields).
```

Comment:
I like your original `ts_A_select_compare_two_fields_not_allowed` suggestion to clarify the failure.